convert2rhel/pkgmanager/handlers/dnf/__init__.py

```diff
@@ -156,14 +162,26 @@ def _resolve_dependencies(self):
             self._base.resolve(allow_erasing=True)
         except pkgmanager.exceptions.DepsolveError as e:
             loggerinst.debug("Got the following exception message: %s" % e)
-            loggerinst.critical("Failed to resolve dependencies in the transaction.")
+            loggerinst.critical_no_exit("Failed to resolve dependencies in the transaction.")
+            raise exceptions.CriticalError(
+                id_="FAILED_TO_RESOLVE_DEPENDENCIES",
+                title="Failed to resolve dependencies.",
+                description="During package transaction convert2rhel failed to resolve the necessary dependencies needed for a package replacement.",
```

Comment:
Not sure that if we should say that _we_ failed to resolve a dependency, since it's yum/dnf job to do that... I think that it may cause some misunderstanding for the user when reading that _we_ tried to do something that is not technically us that are doing it...

Reply:
Changed!